tests/robot-tests/tests/admin/bau/create_methodology_for_publication.robot

```diff
@@ -18,4 +18,14 @@ teardown suite
 Create Methodology for Publication
     [Tags]  HappyPath
     user creates test publication via api   ${PUBLICATION_NAME}
-    user creates methodology for publication    ${PUBLICATION_NAME}
\ No newline at end of file
+    user opens publication on the admin dashboard   ${PUBLICATION_NAME}
```

Comment:
Generally, we should leave two spaces between a keyword and its selector. 

i.e. `user checks page does not contain button  Link to an externally hosted methodology`

Reply:
I'm inclined to think it's ok if we have more than 2 spaces to separate arguments - sometimes it can be clearer if there are more spaces.

The RF docs recommend using four spaces: https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#space-separated-format
`Although using two spaces as a separator is enough, it is recommended to use four spaces to make the separator easier to recognize.`